stacks/java-quarkus/devfile.yaml

```diff
@@ -2,7 +2,11 @@ schemaVersion: 2.0.0
 metadata:
   name: java-quarkus
   version: 1.1.0
-  website: https://quarkus.io
```

Comment:
Do we want to remove the project field from the devfile?

Reply:
I checked the metadata field of API reference: https://docs.devfile.io/devfile/2.1.0/user-guide/api-reference.html and found the website field is not defined under metadata field, that's why I removed that, but I am open to add it back to both this devfile.yaml and API reference as I am gonna create another PR for API reference to add some fields (eg. projectType, language) under metadata.

Reply:
Do you think we should add the website field back to both this devfile.yaml and API reference? Currently in our registry repo, only java-quarkus has the website field under metadata.